cmd/builder/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.23-alpine3.20
+RUN apk --update add ca-certificates
+
+ARG SERVICE_NAME=ocb
+
+RUN addgroup --gid 1001 -S ${SERVICE_NAME} && \
```

Comment:
Can you expand the option names in `addgroup` and `adduser`? The man page for `addgroup` does not even describe an `-S` option, so it's hard to reason about what the command does.

Reply:
Oh I can see now the Alpine docs describe it: https://wiki.alpinelinux.org/wiki/Setting_up_a_new_user. Still, much better to expand the option names for readability.

Reply:
I've expanded the argument names as well as removed an unnecessary argument, good call.